app/sidekiq/hca/ezr_submission_job.rb

```diff
@@ -38,6 +40,22 @@ def self.decrypt_form(encrypted_form)
       JSON.parse(HealthCareApplication::LOCKBOX.decrypt(encrypted_form))
     end
 
+    def self.send_failure_email(parsed_form)
+      email = parsed_form['email']
```

Comment:
Do we always have an email here? I know on the 10-10EZ we have already validated if the email exists by this time. I just wanted to confirm that we know the form has an email.